package.json

```diff
@@ -56,24 +56,24 @@
     "zone.js": "~0.9.1"
   },
   "devDependencies": {
-    "@angular-devkit/core": "^8.0.3",
-    "@angular-devkit/schematics": "^8.0.3",
+    "@angular-devkit/core": "^8.2.0",
+    "@angular-devkit/schematics": "^8.2.0",
     "@angular/bazel": "^8.1.0",
```

Comment:
Note: I wanted to bump the Bazel dependencies as well, but updating `@angular/bazel` causes only the CI to throw an error along the lines of `Cannot find command bazel`. It only happens on the CI and the error wasn't super helpful so I wasn't to track it down. My best guess is that it might be something with the `setup_bazel_binary.sh`.

Reply:
The `@angular/bazel` package should not be related to the `bazel` binary at all. Should we just get this PR through and I'll have a look at the Bazel related thing?

Reply:
Sure.